mne/minimum_norm/time_frequency.py

```diff
@@ -368,8 +493,29 @@ def _source_induced_power(
     verbose=None,
 ):
     """Aux function for source induced power."""
+    if label:
+        _validate_type(
+            label,
+            types=(Label, BiHemiLabel, list, None),
```

Comment:
pretty much anywhere we allow `list` we allow `tuple`, too, so might as well add here